scripts/nodelib.py

```diff
@@ -89,6 +90,12 @@ def check_and_setup(nodocker, is_release, image, home_dir, init_flags, no_gas_pr
           prompt += ": "
         account_id = input(prompt)
         init_flags.append('--account-id=%s' % account_id)
+
+    if chain_id == 'testnet':
+        if os.path.exists('near/res/testnet_records.json.aa'):
```

Comment:
I think check testnet_records.json non-exist would be better, because this .aa will always exist because it's committed to github